### ISSUE TITLE
Claims it is out of bounds but is not

### ISSUE THREAD

**Issue by raacampbell** [BaselLaserMouse/BakingTray#116]
``` 
>> hBT.recipe.tilePattern

** ERROR:
Minimum allowed Y position is -9.00 but tile position array will extend to -9.75
Not returning any tile positions. Try repositioning your sample.

ans =

     []
```

Asking for a smaller tile pattern, then running ` hBT.recipe.tilePattern` then returning to the desired pattern causes the error to go away.

**Comment by raacampbell** [comment]
Happens with Y max/min of +/-9 mm. Not found it with Range of -13 to +3. Odd, since this is 2 mm smaller distance.

**Comment by raacampbell** [comment]
Just had this issue setting up a sample. It happened after I used the "set ventral midline" option. So perhaps it's related to that.

**Comment by raacampbell** [comment]
Yep. Move stage to 0/0, set sample to a brain-sized thing (x=6.4 and y=10.6), hit "set ventral midline" and we see: 

```
** ERROR:
Minimum allowed Y position is -5.40 but tile position array will extend to -10.55
Not returning any tile positions. Try repositioning your sample.
Attempted to make a tile pattern from -6.05 to 0.00 in X and -10.55 to 0.00 in Y
ERROR in setFrontLeftFromVentralMidLine: tile position data are empty. Likely an invalid setting. Can not proceed.
```

**Comment by raacampbell** [comment]
Should be fixed: 4bbf4996adca9c1035f24cf9a0341644535e4110
